[OSS in Business] Created Contribution.md (#526)
todogroup/ospology: whitepapers/business-value/Contribution.md

# Steps to Contribute to OSS Whitepaper! 

> This is basically an Overview of what this document should entail. 

Reviewer: ```suggestion
This document provides an overview of the different ways individuals can contribute as co-creators of the whitepaper, as well as the different roles they can take on.
```

> The Contribution.md file will serve as a guide, providing the needed step by step process of how contributors will collaboratively work on the OSS in business whitepaper, ensuring collective approval from everyone invloved. However given the large scope and extensive work required, I would recommend organising contributors into three groups: They include:

Reviewer: ```suggestion
The Contribution.md file provides a step by step process of how contributors will work on the OSS in business whitepaper, ensuring collective approval from everyone invloved. Contributions can be organized into three groups:
```

> - Group members: Contributors will be organised into groups based off interest and expertise aligned with the topics and section provided through the outline in the whitepaper. Each group will be led by a leader providing the required structure and ensuring accountability for progress.

Reviewer: ```suggestion
- Group participants: Contributors will be organised into groups based on interest and expertise aligned with the topics and section provided through the outline in the whitepaper. Each group will be led by a leader providing the required structure and ensuring accountability for progress.
```

> - Group Lead: A group lead will be responsible for cordinating efforts and facilitating the progress of it's memebers. Group leads will serve beyond just guidance for the contributors as they work on their respective sections before syncing through mini presentations during our calls. However, individuals interested in becoming group leads will have to meet set up basic requirements made by the working group before expressing their interest.  

Reviewer: ```suggestion
- Group Chair: A group chair will be responsible for coordinating efforts and facilitating the progress of the working group participants of each section. Group chairs will serve beyond just guidance for the contributors as they work on their respective sections before syncing through mini presentations during Touchpoint calls. However, individuals interested in becoming group chairs will have to meet set up basic requirements made by the working group before expressing their interest.  
```

> - The reviewer: This is a group of people that will focus more on reviewing the content of other contributors. Their role will ensure that all the work are well documented in google docs used for compling the contriubutions of all groups forming the whitepaper, while ensuring that the outlines and objectives of the working group for the whitepaper are maintained properly --- Additionally, I would also recommend that the working group should make the google doc used for complilations public so viewers can also serve as reviewers through comments and suggestions which will be addressed in the meeting if they cannot be resolved within google docs.

Reviewer: ```suggestion
- Reviewer: This is a group of people that will focus more on reviewing the content of other contributors. Their role will ensure that all the work are well documented in google docs used for compling the contributions of all groups forming the whitepaper, while ensuring that the outlines and objectives of the working group for the whitepaper are maintained properly. The working group should make the google doc used for compilations public so viewers can also serve as reviewers through comments and suggestions which will be addressed in the meeting if they cannot be resolved within google docs. Later, the content should be moved into markdown format and added into this repo via Pull Request.
```

> Please note: I will be open for any questions and clarity if needed. 

Reviewer: ```suggestion
> Please note: I will be open for any questions and clarity if needed. 
```
